### PR TITLE
feature(component): table

### PR DESCRIPTION
**Pull request by dyarmachok**
Ticket: [#42](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/42)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit ffcf04222f09656cb4fdfd30362bc57c1903437e

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cb9c88a90aab80008697bc3

File: components/tables/stories/tables-docs.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Tables
+
+The Mega Menu componet receive an Array called `items` which build the Menu itself
+
+Example:
+
+```
+:items='[
```
Comment (gepifanio) [inline components/tables/stories/tables-docs.md]: Need to update the story, following the usage of tables, and not the Mega Menu 😆

File: components/tables/index.css [hunk 2]
```diff
@@ -125,3 +126,42 @@
     }
   }
 }
+
+@media (--bp-max-x-mobile) {
```
Comment (gepifanio) [inline components/tables/index.css]: Wrapping whole classes can create maintenance issues. If, at a future time a class needs modification on a new breakpoint, you have to duplicate the classes.

Instead, you should nest breakpoints inside each class. This makes it easier to add/remove breakpoints in the future.

.class {
  ...

  @media (--bp-desktop) {
    ...
  }

  @media (--bp-x-desktop) {
    ...
  }
}

**Comment by TimMansell** [comment]
Closing, as we're going to migrate gen1 code over.

**Pull request closed without merging**